### PR TITLE
add docs for lovr.input module

### PR DESCRIPTION
**Pull request by bcampbell**
docs to go with https://github.com/bjornbytes/lovr/pull/22

### CODE REVIEW

File: docs/lovr.input.isMouseDown.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    isDown = lovr.input.isMouseDown(button)
+
+### Arguments
+
+- `MouseButton button` The button to check (`left`, `right` or `middle`)
+
+### Returns
+
+- `boolean isDown` Whether or not the button is currently pressed.
```
Comment (cvan) [inline docs/lovr.input.isMouseDown.md]: nit: there's no closing period in the other places

**Comment by cvan** [comment]
just saw this when going through this repo. (sorry for the drive-by comment!) this is great to add! 👍 

P.S. your GitHub avatar is fantastic. :penguin:

**Comment by bcampbell** [comment]
Afraid to say that the input API stuff I did isn't being merged. I instead did some work to abstract out the headset code and provide a fake mouseview headset when there's no real VR headset available (and that stuff has been merged in).

I think it was the right call to leave out my half-baked input API for now. I'd definitely like to see a proper input system added some point down the line (mainly because I think lovr would be a great 3D toolkit for non-VR projects too), but for now I think keeping the focus narrow and on VR is the way to go.

(thanks for the nice comment about my avatar - it was me dog-fooding my [evilpixie](http://evilpixie.scumways.com/), my pixel-art program!)

**Comment by bjornbytes** [comment]
Woops!  Yeah this should be closed.

**Pull request closed without merging**